media-plugins/distrho-ports/distrho-ports-99999999.ebuild

```diff
@@ -22,41 +22,26 @@ REQUIRED_USE="|| ( lv2 vst )"
 
 RDEPEND="media-libs/alsa-lib
 	media-libs/freetype
-	media-libs/ladspa-sdk
```

Comment:
I don't really understand the situation with LADSPA but it seems like distrho-ports doesn't build LADSPA plugins anymore although there is mention of LADSPA stuff throughout the codebase. I'll ask Falk about this.

Reply:
Got an answer from Falk, there's no way to build LADSPA plugins, it's just references from the included Juce code.